memory.c

```diff
@@ -27,13 +27,22 @@
 
 void memory_protect(void)
 {
+	// Reference STM32F205 Flash programming manual revision 5 http://www.st.com/resource/en/programming_manual/cd00233952.pdf
+	// Section 2.6 Option bytes
 	//                     set RDP level 2                   WRP for sectors 0 and 1
 	if ((((*OPTION_BYTES_1) & 0xFFFF) == 0xCCFF) && (((*OPTION_BYTES_2) & 0xFFFF) == 0xFFFC)) {
```

Comment:
No, I don't think so based on table 7 in section 2.6.1. It shows bit 4 of the user byte being 1 and bits 15-12 of the other as being 0xF. Do you read that table differently?